lib/resources/checks.js

```diff
@@ -21,6 +21,10 @@ util.inherits(Checks, ResourceBase);
     return this._transmit('GET', null, options, null, callback);
   };
 
+  this.delete = function (id, callback) {
+    return this._transmit('DELETE', id, null, null, callback);
+  }
```

Comment:
I actually forgot to run eslint before committing, but running it now see my above comment